docker/openproblems-r-extras/r_requirements.txt

```diff
@@ -1,4 +1,10 @@
 bioc::batchelor
+bioc::ComplexHeatmap
+sparsesvd
+Seurat
+conos
+rliger
```

Comment:
do we need/want to pin package versions here or do you think this will be future proof? If likely to work, let's keep it this way and version open problems outputs instead.

Reply:
good question, I thought the idea was to keep the results up to date? In this case I would only pin or adapt the code once something breaks